docs/_plugins/search_index.rb

```diff
@@ -158,6 +158,27 @@ def benchmarking_results(post_url)
     nil
   end
 
+  def references_results(post_url)
+    if @content.include? '## References'
+      m = /^## References[^#]+([\W\w]*?)($)/m.match(@content)
+      if m
+        references_section = m[0]
+        url_scans = references_section.scan(URI.regexp)
+        return url_scans.map do |url_parts|
+                    if not url_parts.one?
+                      url_parts.each_with_index.map do |part, index|
+                        if index == 0
+                          part = part + "://"
+                        end
+                        part
+                      end.join.delete_suffix(')')
```

Comment:
Could we remove `.` as well?
It will prevent creating URLs like:
https://fasttext.cc/docs/en/crawl-vectors.html).
https://wacky.sslmit.unibo.it/doku.phpid=corpora).

Reply:
Done